cmd/queue/main.go

```diff
@@ -214,8 +215,57 @@ func createAdminHandlers() *http.ServeMux {
 	return mux
 }
 
+func execHTTPAction(url string) error {
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+
+	if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusBadRequest {
+		return fmt.Errorf("HTTP request failed with status: %v", resp.StatusCode)
+	}
+
+	return nil
+}
+
+func buildAdminUrl(path string) string {
+	url :=  url.URL{
+		Scheme: "http",
+		Host:   fmt.Sprintf("localhost:%d", v1alpha1.RequestQueueAdminPort),
+		Path:   path,
+	}
+
+	return url.String()
+}
+
+func handleActions() (bool, error) {
+	var err error
+
+	switch flag.Arg(0) {
+	case queue.CommandModeExec:
+		if len(flag.Args()) > 1 {
+			actionMap := map[string]string{
+				queue.ActionQuit:   buildAdminUrl(queue.RequestQueueQuitPath),
+				queue.ActionHealth: buildAdminUrl(queue.RequestQueueHealthPath),
+			}
+
+			action := flag.Arg(1)
+			if url, ok := actionMap[action]; ok {
+				return true, execHTTPAction(url)
+			}
+
+			err = errors.New("unknown exec action")
```

Comment:
I think it'd be more readabble, if you just:
```
if len() {
 ...
 return false, errors.New()
}
return false, errors.New()
}
...
return false, nil
```
Then you won't need the `err` variable at all.

Reply:
Done!